### PR TITLE
BIZ-512 fixed playing video

### PR DESCRIPTION
**Pull request by RomanFedytskyi**

### CODE REVIEW

File: dev/library/ctrl.js [hunk 1]
```diff
@@ -69,6 +69,7 @@
     //Preview popup
     vm.showPreview = function (video) {
       var ext = video.attributes.external_file_name.split('.').pop();
+      ext = (ext === 'wmv') ? 'webm' : ext;
```
Comment (dmitrij-borchuk) [inline dev/library/ctrl.js]: Why?

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: dev/library/ctrl.js (hunk 1 above)
Comment (RomanFedytskyi) [reply dev/library/ctrl.js]: Because video tag don't have type video/wmv, only webm, ogg, mp4

Comment (dmitrij-borchuk) [reply dev/library/ctrl.js]: Does it mean that we can't play wmv?

Comment (RomanFedytskyi) [reply dev/library/ctrl.js]: every browser does not support WMV as a type of file. Aside from IE, this type of format is unlikely to be compatible with most browsers.

Comment (dmitrij-borchuk) [reply dev/library/ctrl.js]: So how changing 'wmv' to 'webm'  helps us?

**Review by dmitrij-borchuk — APPROVED** [review]

**Pull request merged**